SECURITY.md

```diff
@@ -2,7 +2,7 @@
 
 ## Supported Versions
 
-Please see [Releases](https://github.com/handshake-org/hsd/releases).
+Please see [Releases](https://github.com/handshake-org/hsd/releases).  
```

Comment:
nit: extra whitespace?

Reply:
New line basically. Just new line wont render text to the next line without trailing spacing in the previous one.